HEN_HOUSE/egs++/geometry/egs_rz/rz1.geom

```diff
@@ -0,0 +1,18 @@
+:start geometry definition:
+
+    :start geometry:
+        name = phantom
+        library = egs_rz
+        radii =  1, 2, 3
+        z-planes = -4 -3 -2 -1 0 1 2 3 4
+        :start media input:
+            media = M0
+        :stop media input:
+    :stop geometry:
+
+
+    simulation geometry = phantom
+
+
+:stop geometry definition:
+
```

Comment:
This file is not needed - there is already an example in `examples/rz_lib1.geom`.